assets/css/shared.scss

```diff
@@ -0,0 +1,201 @@
+/**
```

Comment:
The css here is loaded twice in the editor since it's built into both `style.css` & `editor.css`, this can just be put into `style.scss`